ste/sender-blockBlob.go

```diff
@@ -273,6 +273,32 @@ func (s *blockBlobSenderBase) Cleanup() {
 	}
 }
 
+//Currently we've common Metadata Copier across all senders for block blob.
+func (s *blockBlobSenderBase) GenerateCopyMetadata(id common.ChunkID) chunkFunc {
+	return createChunkFunc(true, s.jptm, id, func() {
+		if s.jptm.Info().PreservePOSIXProperties {
```

Comment:
❓ It feels relevant to implicitly flip PreservePOSIXProperties should a user specify to overwrite POSIX properties.